app/models/time_sheet.rb

```diff
@@ -1,10 +1,37 @@
+# == Schema Information
+#
+# Table name: time_sheets
+#
+#  id                    :integer          not null, primary key
+#  month                 :integer
+#  year                  :integer
+#  salary                :integer
+#  salary_is_per_month   :boolean
+#  workload              :integer
+#  workload_is_per_month :boolean
+#  user_id               :integer
+#  project_id            :integer
+#  created_at            :datetime         not null
+#  updated_at            :datetime         not null
+#  signed                :boolean          default(FALSE)
+#  handed_in             :boolean          default(FALSE)
+#  last_modified         :date
+#  status                :integer          default(0)
+#  signer                :integer
+#
+
 class TimeSheet < ActiveRecord::Base
     belongs_to :user
     belongs_to :project
+    enum status: [ :pending, :accepted, :rejected]
 
     validates :workload_is_per_month, :inclusion => { :in => [true, false] }
     validates :salary_is_per_month, :inclusion => { :in => [true, false] }
 
+    def show_add_signature_prompt
+      flash[:error] = 'Please add signature'
```

Comment:
Das kommt nicht von uns, wir mussten aber leider den Branch https://github.com/hpi-swt2/wimi-portal/tree/feature/48_stundenzettel_bestaetigen in unseren mergen, da wir davon abhängen und er zum selben Sprint fertig sein soll